packages/next-ui/Snackbar/MessageSnackbarImpl.tsx

```diff
@@ -54,8 +60,9 @@ export default function MessageSnackbarImpl(props: MessageSnackbarProps) {
     sticky,
     children,
     onClose,
-    severity = 'info',
+    severity = 'success',
```

Comment:
Maybe we should change it to 'info', and check usage sights and add 'success' there? Better to have an info icon (the one you added) by default I think :)